### PR TITLE
Naming convention changes

### PR DESCRIPTION
**Pull request by augustocollado**
### Description

This PR applies the new naming convention to every article and diagram.
It changes:
ConatainerChain & Apppchain -> appchains (only capitalized when in titles or beginning the sentence)
Collators -> block producers

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one
- [x] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

File: builders/build/customize/prerequisites.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Prerequisites for building an Appchain
-description: Install the basic set of tools and software to set up a local development environment and be able to compile, run, and test your Appchain.
+title: Prerequisites for building an appchain
```
Comment (eshaben) [inline builders/build/customize/prerequisites.md:2]:
```suggestion
title: Prerequisites for Building an Appchain
```

File: builders/build/templates/evm.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ This means that this template is ready to be built as-is and deployed through Ta
 
 ## Included Modules {: #included-modules }
 
-Tanssi EVM Appchain template is built on top of the [Substrate Appchain template](/builders/build/templates/substrate/){target=\_blank}. Consequently, it includes the same [baseline modules](/builders/build/templates/substrate/#included-modules).
+Tanssi EVM appchain template is built on top of the [Substrate Appchain template](/builders/build/templates/substrate/){target=\_blank}. Consequently, it includes the same [baseline modules](/builders/build/templates/substrate/#included-modules).
```
Comment (eshaben) [inline builders/build/templates/evm.md:33]:
```suggestion
The Tanssi EVM appchain template is built on top of the [Substrate Appchain template](/builders/build/templates/substrate/){target=\_blank}. Consequently, it includes the same [baseline modules](/builders/build/templates/substrate/#included-modules).
```

File: builders/build/templates/overview.md [hunk 3]
```diff
@@ -11,37 +11,37 @@ Appchains deployed through Tanssi are essentially parachains within the Polkadot
...
-- **[XCM](https://wiki.polkadot.network/docs/learn-xcm){target=\_blank}** - handles the ingestion and dispatch of incoming downward and lateral messages, allowing a Tanssi  Appchain to communicate and interoperate with the other sibling chains within the ecosystem
-- **Runtime Upgrades** - a runtime upgrade in a Tanssi Appchain must be informed to the relay chain to allow its validators to check on the blocks produced by the collators of the Tanssi Appchains. Cumulus notifies the upgrade to the relay chain and waits the required amount of time (blocks) before enacting the change
+- **Consensus** - Cumulus adds the necessary functionality to allow the block producers to produce blocks, gossip and validate them, and coordinate with the relay chain to get notified about the block's finality
+- **[XCM](https://wiki.polkadot.network/docs/learn-xcm){target=\_blank}** - handles the ingestion and dispatch of incoming downward and lateral messages, allowing a Tanssi  appchain to communicate and interoperate with the other sibling chains within the ecosystem
```
Comment (eshaben) [inline builders/build/templates/overview.md:28]:
```suggestion
- **[XCM](https://wiki.polkadot.network/docs/learn-xcm){target=\_blank}** - handles the ingestion and dispatch of incoming downward and lateral messages, allowing a Tanssi appchain to communicate and interoperate with the other sibling chains within the ecosystem
```

File: builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md [hunk 4]
```diff
@@ -46,26 +46,26 @@ Your tokens will be disbursed shortly, and you can verify your TANGO token balan
...
 
 === "Ethers.js"
 
     ```js
     import { ethers } from "ethers";
 
     const providerRPC = {
-      EvmContainer: {
-        name: 'dancebox-evm-container',
+      EvmAppchain: {
```
Comment (eshaben) [inline builders/tanssi-network/networks/dancebox/demo-evm-containerchain.md:57]: This should actually be `evmAppchain`

File: builders/tooling/oracles/acurast.md [hunk 5]
```diff
@@ -131,7 +131,7 @@ To configure your job, head to the [Acurast console](https://console.acurast.com
 2. Select **Moonbeam** as the chain
 3. Select **Moonbase** as the environment. Remember that Tanssi's EVM-compatibility is derived from Moonbeam
 4. Select **Price Feeds**
-5. Paste in the code of your job script. You can copy and paste directly from the [script of the sample BTC/USD price feed](https://github.com/Acurast/acurast-evm-oracle-sample/blob/main/acurast_scripts/oracle_job.js){target=\_blank}, just make sure to change the destination contract to one that you deployed on your Appchain and the RPC node to your [Appchain's RPC URL](https://apps.tanssi.network/)
+5. Paste in the code of your job script. You can copy and paste directly from the [script of the sample BTC/USD price feed](https://github.com/Acurast/acurast-evm-oracle-sample/blob/main/acurast_scripts/oracle_job.js){target=\_blank}, just make sure to change the destination contract to one that you deployed on your appchain and the RPC node to your [Appchain's RPC URL](https://apps.tanssi.network/)
```
Comment (eshaben) [inline builders/tooling/oracles/acurast.md:134]:
```suggestion
5. Paste in the code of your job script. You can copy and paste directly from the [script of the sample BTC/USD price feed](https://github.com/Acurast/acurast-evm-oracle-sample/blob/main/acurast_scripts/oracle_job.js){target=\_blank}, just make sure to change the destination contract to one that you deployed on your appchain and the RPC node to your appchain's RPC URL, which can be found on the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}
```

File: learn/tanssi/overview.md [hunk 6]
```diff
@@ -1,52 +1,52 @@
...
 
-While a block producer simultaneously executes blocks from Tanssi and the Appchain that was assigned to it, Tanssi and each Appchain will have their own set of Data Preservers running full archive nodes for each chain. These Data Preservers will provide the RPC infrastructure for apps and users interacting with Tanssi and Appchains.
+While a block producer simultaneously executes blocks from Tanssi and the appchain that was assigned to it, Tanssi and each appchain will have their own set of Data Preservers running full archive nodes for each chain. These Data Preservers will provide the RPC infrastructure for apps and users interacting with Tanssi and appchains.
 
-![Data Preservers of an Appchain & Tanssi](/images/learn/tanssi/overview/dark-overview-2.webp#only-dark)
-![Data Preservers  of an Appchain & Tanssi](/images/learn/tanssi/overview/light-overview-2.webp#only-light)
\ No newline at end of file
+![Data Preservers of an appchain & Tanssi](/images/learn/tanssi/overview/dark-overview-2.webp#only-dark)
+![Data Preservers  of an appchain & Tanssi](/images/learn/tanssi/overview/light-overview-2.webp#only-light)
```
Comment (eshaben) [inline learn/tanssi/overview.md:52]:
```suggestion
![Data Preservers of an appchain & Tanssi](/images/learn/tanssi/overview/light-overview-2.webp#only-light)
```

**Review by eshaben — CHANGES_REQUESTED** [review]
I did a search for 'Appchain' with a capitalized A in visual studio code, and came across some files that still need to be updated - mostly in comments in code snippets, but also a few text snippets, a few meta-descriptions, the README.md, and scattered throughout some of the newer files (maybe just missed do to merging at different times - not sure)

Also did a search for "Collators" - the node-operators/index.md file needs to be updated (title and description), the alt text for image 3 in learn/tanssi/technical-features.md needs to be updated

There are also a few snippets that use "evm_containerchain" as a variable, this can be changed to "evm_appchain"

"ContainerChain" when searched also came back with a few hits - mostly on index.md pages, on the CONTRIBUTING.md page, and also one time in a comment on a snippet (feeder.ts)

While we are applying these conventions, are there specifications outlined for "Dedicated appchains" and "Snap appchains" should dedicated and snap be capitalized?

**Comment by augustocollado** [comment]
There are no definitions yet for dedicated or snaps
There were still lots of remaining references to deprecated terms, thanks :)

> I did a search for 'Appchain' with a capitalized A in visual studio code, and came across some files that still need to be updated - mostly in comments in code snippets, but also a few text snippets, a few meta-descriptions, the README.md, and scattered throughout some of the newer files (maybe just missed do to merging at different times - not sure)
> 
> Also did a search for "Collators" - the node-operators/index.md file needs to be updated (title and description), the alt text for image 3 in learn/tanssi/technical-features.md needs to be updated
> 
> There are also a few snippets that use "evm_containerchain" as a variable, this can be changed to "evm_appchain"
> 
> "ContainerChain" when searched also came back with a few hits - mostly on index.md pages, on the CONTRIBUTING.md page, and also one time in a comment on a snippet (feeder.ts)
> 
> While we are applying these conventions, are there specifications outlined for "Dedicated appchains" and "Snap appchains" should dedicated and snap be capitalized?

File: builders/interact/ethereum-api/indexers/subsquid.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Index Your Tanssi EVM AppChain with Subsquid
```
Comment (eshaben) [inline builders/interact/ethereum-api/indexers/subsquid.md:2]:
```suggestion
title: Index Your Tanssi EVM Appchain with Subsquid
```

**Review by eshaben — COMMENTED** [review]
Sweeeet! One last fix

**Pull request merged**